.github/workflows/gen-docs.yml

```diff
@@ -0,0 +1,36 @@
+name: Generate Documentation
+
+on: [release]
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Just making sure, since I can't look into the repos settings. We use the default `GITHUB_TOKEN`? As described [here](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/authenticating-with-the-github_token#about-the-github_token-secret). No extra GITHUB_TOKEN has been set in the settings?

Reply:
this the default GITHUB_TOKEN